dashmips/debuggerserver.py

```diff
@@ -65,23 +45,40 @@ def debug_mips(program: MipsProgram, host="localhost", port=2390, should_log=Fal
         format="%(asctime)-15s %(levelname)-7s %(message)s",
         level=log.INFO if should_log else log.CRITICAL,
     )
-    logger = log.getLogger("websockets.server")
+    logger = log.getLogger("sockets.server")
     logger.addHandler(log.StreamHandler())
-    log.info(f"Serving on: ws://{host}:{port}")
-
-    # Collect functions from debugger.py
-    debugger_module = importlib.import_module(".debugger", "dashmips")
-    funcs = inspect.getmembers(debugger_module, inspect.isfunction)
-    commands = {}
-    for name, command in funcs:
-        commands[name.replace("debug_", "")] = functools.partial(command, program=program)
-
-    ws_func = functools.partial(dashmips_debugger, commands=commands)
-    start_server = websockets.serve(ws_func, host, port, close_timeout=2000)
-    loop = asyncio.get_event_loop()
-    try:
-        ws_server = asyncio.get_event_loop().run_until_complete(start_server)
-        loop.run_forever()
-    except KeyboardInterrupt:
-        log.warning('Shutting down debugger...')
-        loop.close()
+    log.info(f"Serving on: s://{host}:{port}")
+
+    class DashmipsTCPServerHandler(socketserver.BaseRequestHandler):
+        def setup(self):
+            # Collect functions from debugger.py
+            debugger_module = importlib.import_module(".debugger", "dashmips")
+            funcs = inspect.getmembers(debugger_module, inspect.isfunction)
+            self.commands = {}
+            for name, command in funcs:
+                self.commands[name.replace("debug_", "")] = functools.partial(command, program=program)
+
+        def handle(self):
+            # self.request is the TCP socket connected to the client
+            while True:
+                header = b""
+                while True:
+                    header += self.request.recv(1)
+                    if header and chr(header[-1]) == '}':
+                        break
+
+                msg_size = int(header[8:-1])
+                command = self.request.recv(msg_size)
+
+                print("{} wrote:".format(self.client_address[0]))
+                print(command)
```

Comment:
lets search and replace `print`s with `log.info` or `log.debug`, this prevents clutter on the terminal when someone runs dashmips without the `-l` flag